gulpfile.js

```diff
@@ -1,7 +1,19 @@
 var eslint = require('gulp-eslint');
 var gulp = require('gulp');
+var isparta = require('isparta');
+var istanbul = require('gulp-istanbul');
+var mocha = require('gulp-mocha');
 
-gulp.task('lint', function() {
+gulp.task('lint-test', function() {
```

Comment:
Is there really much benefit to splitting the lint into test and lib?
Seems like we'd do fine with a single task.